app/controllers/jobseekers/job_applications_controller.rb

```diff
@@ -66,7 +68,7 @@ def review_form_params
 
   def vacancy
     @vacancy ||= if params[:job_id].present?
-                   Vacancy.live.find(params[:job_id])
+                   Vacancy.find(params[:job_id])
```

Comment:
This change would mean I can start an application for an expired job

Reply:
You're right - I forgot to get rid of this after making the expired check only happen on the final submit.